Changes

```diff
@@ -1,5 +1,24 @@
 Release history of Zonemaster
 
+v2023.1.3 2023-08-07 (public fix version)
+
+ [Fixes]
+ - Fixes bug that made all tests of zone names that starts with the same
+   letters as the whole first label of the parent zone fail, e.g.
+   "NOrid.NO" and "FRance.FR". See Zonemaster-Engine.
```

Comment:
same suggestion as in https://github.com/zonemaster/zonemaster-engine/pull/1271#discussion_r1285510401: add the fact that this is a regression, there was no such bug prior to v2023.1.1